projects/laji/test/+user/user.po.ts

```diff
@@ -0,0 +1,50 @@
+import { Page, expect } from '@playwright/test';
+
+// use this when page is expected to be on the laji-auth page
+export const lajiAuthLogin = async (page: Page) => {
+  await page.locator('#local-login').click();
+  await page.locator('[name="email"]').fill(process.env.E2E_USER);
+  await page.locator('[name="password"]').fill(process.env.E2E_PASS);
+  await page.locator('button.submit').click();
+};
```

Comment:
The login is causing flakiness still. The expect timeout is 5s, so when a test case calls `lajiAuthLogin()` and starts doing assertions, the first assertion will run into that timeout because the login submit can take longer. Maybe this method should wait for the login submit to go through, so the tests using this don't have to care about that?

Se we could add something like this to the bottom after the submit click:

```await page.locator('#local-login').waitFor({ state: 'detached', timeout: 30 * 1000 })```

Btw maybe use JS doc notation for the comment ("use this when...") since it could be useful to have it as a method description in an IDE? `/**`

Reply:
The permanent token should fix these issues I guess? So idk maybe no need to spend too much effort on this method right now, as we might later ditch it anyway?